tests/foreman/ui/test_operatingsystem.py

```diff
@@ -118,49 +81,71 @@ def test_positive_end_to_end(session):
         assert len(os['parameters']['os_params']) == 1
         assert os['parameters']['os_params'][0]['name'] == param_name
         assert os['parameters']['os_params'][0]['value'] == param_value
+        template_name = gen_string('alpha')
```

Comment:
I think OS created will have DEFAULT_TEMPLATE assigned as per old tests, so can this be asserted after create? and then update template assigned?

Reply:
Updated